### PR TITLE
refactor: VIDEO→VIDEO_LEGACY, FRAME→VIDEO

### PR DESCRIPTION
**Pull request by PierreLeveau**
fixes the CI

### CODE REVIEW

File: src/kili/enums.py [hunk 1]
```diff
@@ -175,7 +175,7 @@
 
 
 UploadType = Literal[
-    "VIDEO",
+    "VIDEO_LEGACY",
```
Comment (theodu) [inline src/kili/enums.py:178]: This should stay as "VIDEO". It is an enum that has two type for asynchronous upload: VIDEO or GEO_SATELLITE

File: src/kili/mutations/asset/__init__.py [hunk 2]
```diff
@@ -58,16 +58,16 @@ def append_many_to_dataset(
                 - For a `TEXT` project, the content can be either raw text, or URLs to TEXT assets.
                 - For an `IMAGE` / `PDF` project, the content can be either URLs or paths to existing
                     images/pdf on your computer.
-                - For a `VIDEO`  project, the content must be hosted on a web server,
+                - For a `VIDEO_LEGACY` project, the content must be hosted on a web server,
```
Comment (theodu) [inline src/kili/mutations/asset/__init__.py:61]: 👍

File: src/kili/mutations/asset/__init__.py [hunk 3]
```diff
@@ -58,16 +58,16 @@ def append_many_to_dataset(
                 - For a `TEXT` project, the content can be either raw text, or URLs to TEXT assets.
                 - For an `IMAGE` / `PDF` project, the content can be either URLs or paths to existing
                     images/pdf on your computer.
```
Comment (theodu) [inline src/kili/mutations/asset/__init__.py:60]: - For a `VIDEO` project, the content can be either URLs pointing to videos hosted on a web server or paths to existing video files on your computer. If you want to import video from frames, look at the json_content section below

File: src/kili/services/asset_import/__init__.py [hunk 4]
```diff
@@ -18,7 +18,7 @@
     "IMAGE": ImageDataImporter,
     "TEXT": TextDataImporter,
     "VIDEO": VideoDataImporter,
-    "FRAMES": VideoDataImporter,
+    "VIDEO_LEGACY": VideoDataImporter,
```
Comment (theodu) [inline src/kili/services/asset_import/__init__.py:21]: I admit that when coding the new importer, I didn't even think to the VIDEO_LEGACY input_type. I would have to see if it works with the VideoDataImporter!

File: src/kili/services/asset_import/base.py [hunk 5]
```diff
@@ -153,7 +153,7 @@ def _async_import_to_kili(self, assets: List[KiliResolverAsset]):
         """
         Import assets with asynchronous resolver.
         """
-        upload_type = "GEO_SATELLITE" if self.input_type == "IMAGE" else "VIDEO"
+        upload_type = "GEO_SATELLITE" if self.input_type == "IMAGE" else "VIDEO_LEGACY"
```
Comment (theodu) [inline src/kili/services/asset_import/base.py:156]: keep VIDEO

File: src/kili/services/asset_import/legacy.py [hunk 6]
```diff
@@ -106,7 +106,7 @@ def encode_object_if_not_url(content, input_type):
 
 def process_frame_json_content(json_content):
     """
-    Function to process individual json_content of VIDEO projects
+    Function to process individual json_content of VIDEO_LEGACY projects
```
Comment (theodu) [inline src/kili/services/asset_import/legacy.py:109]: keep VIDEO

File: src/kili/services/asset_import/legacy.py [hunk 7]
```diff
@@ -150,7 +150,7 @@ def process_json_content(
         if is_url(json_content):
             processed_json_content_array.append(json_content)
         else:
-            if input_type in ("FRAME", "VIDEO"):
+            if input_type in ("VIDEO_LEGACY", "VIDEO"):
```
Comment (theodu) [inline src/kili/services/asset_import/legacy.py:153]: this should be only VIDEO.
But I think that we can delete the legacy importer now. We will recode the one from time series when prioritized later ?

File: src/kili/services/asset_import/video.py [hunk 8]
```diff
@@ -1,5 +1,5 @@
 """
-Functions to import assets into a VIDEO project
+Functions to import assets into a VIDEO_LEGACY project
```
Comment (theodu) [inline src/kili/services/asset_import/video.py:2]: VIDEO alone I think

Comment (theodu) [reply src/kili/services/asset_import/video.py:2]: and everywhere below

File: tests/services/asset_import/test_import_common.py [hunk 9]
```diff
@@ -22,14 +22,14 @@ class TestContentType(ImportTestCase):
     @patch.object(
         QueriesProject,
         "projects",
-        MagicMock(return_value=[{"inputType": "VIDEO"}]),
+        MagicMock(return_value=[{"inputType": "VIDEO_LEGACY"}]),
```
Comment (theodu) [inline tests/services/asset_import/test_import_common.py:25]: keep VIDEO

File: tests/services/asset_import/test_import_video.py [hunk 10]
```diff
@@ -108,7 +108,7 @@ def test_upload_one_local_video_to_frames(self):
             ["local video to frames"],
             ["unique_id"],
             [expected_json_metadata],
-            "VIDEO",
+            "VIDEO_LEGACY",
```
Comment (theodu) [inline tests/services/asset_import/test_import_video.py:111]: keep VIDEO

Comment (theodu) [reply tests/services/asset_import/test_import_video.py:111]: and everywhere below, VIDEO_LEGACY was in fact not tested at all!

**Comment by PierreLeveau** [comment]
I'll merge this PR because it fixes the e2e tests, but let's keep the investigation open to know we are not missing anything with this modification.

**Pull request merged**